app/utils/validators/FormFieldValidator.scala

```diff
@@ -68,10 +83,11 @@ object FormFieldValidator {
       case _              => false
   }
 
-  val isAllCapitalLetter: String => Boolean = (input: String) => input.matches(allCapitalLettersRegex)
-
   val isDecimal: String => Boolean = (input: String) => input.matches(decimalRegex)
 
```

Comment:
The method `isDecimalNoLongerThan` is not implemented corretly, according to its name at least.
For now, it is checking if the String contains no more digits than stated. But it may contain hundreads of letters.